Calculator.java

```diff
@@ -0,0 +1,26 @@
+package Tema8;
+
+public class Calculator implements MathOperations<Double> {
+    @Override
+    public Double addition(Double num1, Double num2)throws DivideByZero {
+        return num1+num2;

```

Comment:
In general ar trebuie sa lasi un spatiu intre parametri si opetor.E mai readable asa
`num1+num2`
vs
`num1 + num2`

Dar nu face asta manual, trebuie sa iti setezi din eclipse setarile de formatare si la fiecare save iti formateaza el codul singur si adauga saptiile care lipsesc